pkgs/top-level/php-packages.nix

```diff
@@ -160,7 +160,7 @@ in {
       checkPhase = ''
         runHook preCheck
 
-        NO_INTERACTION=yes SKIP_PERF_SENSITIVE=yes make test
+        NO_INTERACTION=yes SKIP_PERF_SENSITIVE=yes make test TESTS="--offline"
```

Comment:
See also https://github.com/NixOS/nixpkgs/pull/337665, the proper env var should be  `SKIP_ONLINE_TESTS` IIRC.